elements/noflo-component-editor.html

```diff
@@ -87,6 +87,8 @@
         var self = this;
         var _ = require('underscore');
         this.debouncedComponentChange = _.debounce(function () {
+          delete self.component.icon;
```

Comment:
Why this?

Reply:
@jonnor just as a clean-up

Reply:
Clean up what? I don't understand why it is neccesary, and why it does not remove data that seemingly should be preserved...

Reply:
ok, I'll drop that